src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  padding-top: 32px;
+
+  &__container {
+    padding: 0 16px 16px;
+  }
+
+  &__img {
+    background-image: url('../images/imac.jpeg');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 1.5;
+    font-weight: 500;
+    color: $main-accent;
+
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 1.4;
+    font-weight: 400;
+    color: $secondary;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+    color: $main-accent;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 1.5;
+    color: $secondary;
+  }
+
+  &__amount {
+    font-size: 16px;
+    line-height: 1.13;
+    font-weight: 700;
+    color: $main-accent;
+  }
+
+  &__button {
+    font-size: 14px;
+    line-height: 1.14;
```

Comment:
I guess you don't really need to add `line-height` anywhere here in styles

Reply:
> I guess you don't really need to add `line-height` anywhere here in styles

i don't think so, cuz without line-height i can't pass the test and it's also affect to pixel-perfect layout